vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -356,29 +328,25 @@ the flexibility to apply the [`Filter`][vizro.models.Filter] to only one specifi
 
         [FirstPage4]: ../../assets/tutorials/dashboard/dashboard24.png
 
-Fantastic job! By completing your first dashboard page, you have gained valuable skills in creating a
-[`Page`][vizro.models.Page] with multiple `components`, arranging them using layout  configuration, and adding
-functionality to your dashboard.
+Fantastic job! You have completed first dashboard page and gained valuable skills to [create an initial figure on a dashboard page](#2-create-a-first-dashboard-page), [add extra components](#22-add-further-components), [arrange them in a layout configuration](/#23-configure-the-layout), and [set up an interactive dashboard control](#24-add-a-control-for-dashboard-interactivity).
 
-### 3. Create second dashboard page
+## 3. Create a second dashboard page
```

Comment:
@maxschulz-COL @huong-li-nguyen This is the section that I modified -- please could you give it a readthrough to make sure I haven't including anything incorrect? It was missing some explanation and there was a "bug" whereby the code changed the opacity of both charts but the text said it changed only 1. I changed the code. 

Could you confirm all is OK in section 3 please?

Reply:
Looks good to me - have one rephrasing suggestion and I think you just need to copy over your change to the other code tab as well. Currently i think they are out of sync.

I am even wondering why we have these two tabs in the first place? 😅  No need to change it now, but i think we need only one and then add more notes and highlights where the relevant code changes are

Reply:
The idea is that people can follow along more easily by always having both:
- all the code
- and the incremental change